plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
I switched the `generate_scale_file` function to use the local `Procfile` extracted by this step, so it is used in either case. I figured it'd be the best thing since it was being extracted by that command anyway but thrown away.